proposals/0085-additional-fee-collector-constraints.md

```diff
@@ -0,0 +1,90 @@
+---
+simd: "0085"
+title: Additional Fee Collector Constraints
+authors:
+  - Justin Starry
+category: Standard
+type: Core
+status: Draft
+created: 2023-11-05
+feature: https://github.com/solana-labs/solana/issues/33888
+---
+
+## Summary
+
+Burns fees if they would otherwise be collected into an account that violates
+one of the following constraints:
+
+1. Must be a system owned account
+2. Must be rent-exempt
+
+This applies to both transaction fees and rent fees. Note that the "rent-exempt"
+constraint was already added for rent fee collection in
+[Feature 30151](https://github.com/solana-labs/solana/issues/30151).
+
+## Motivation
+
+1. Fee collection occurs outside of the transaction runtime because the Solana
+   protocol mandates that fees are collected at the end of each block. By
+   restricting fee collector accounts to be system owned, the number of account
```

Comment:
Hmm, while I realize we call the accounts `fee_collector` accounts in code (and it's defined as a term below), the word "collected" feels problematic here, because to someone who hasn't read that stuff yet, it implies reaping as well as distributing. (The reaping (debiting) explicitly happens during transaction processing not at the end of the block.) How about this instead?

```suggestion
1. Fee distribution occurs outside of the transaction runtime because the Solana
   protocol mandates that fees are distributed to "fee collector" accounts at the end of each block. By
   restricting fee collector accounts to be system owned, the number of account
```

Reply:
Yeah that's much clearer, thanks!